packages/load-dist/index.js

```
@@ -0,0 +1,14 @@
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/load-dist
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+'use strict';
+
+const path = require('path');
+
+module.exports = function loadDist(projectRootDir) {
+  const nodeMajorVersion = +process.versions.node.split('.')[0];
+  const dist = nodeMajorVersion >= 10 ? './dist10' : './dist8';
```

I think we should throw an error if the Node.js version is lower than 8.